nodes/Postiz/Postiz.node.ts

```
@@ -73,6 +73,12 @@ export class Postiz implements INodeType {
 						description: 'Upload a file to Postiz',
 						action: 'Upload a file to postiz',
 					},
+					{
+						name: 'Upload File from URL',
```

ESLint

[error] Change to title case [autofixable]

(n8n-nodes-base/node-param-display-name-miscased)